docs/src/modules/components/overview/Keyboard.tsx

```diff
@@ -316,21 +316,21 @@ export function KeyboardSvg({ selectedKey, handleKeySelection }: KeyboardSvgProp
                         selectedKey.key.toLowerCase() === label.toLowerCase() &&
                         selectedKey.location === location,
                     })}
-                    onMouseDown={(e) => {
+                    onMouseDown={(event) => {
```

Comment:
Nitpick: The code for these demos is not visible anywhere in the docs, so, it's not a visible problem. 👍

Reply:
It depends, I could fix https://github.com/mui/material-ui/pull/43361/files because I could get a hit for my search of "event.key.toLowerCase() === 'c'" in the codebase.